galaxy_wrappers/CollectResults/glob_res.pl

```diff
@@ -12,6 +12,7 @@
 my $CI                   = 1;
 my $final_partition_soft = "";
 my $part_cmsearch        = "";
+my $combined_cm          = "combined.cm";
```

Comment:
Is combined_cm the aggregated cm-search results? Could you please rename the file, specially the extension? `.cm` is known to used for Covariance Model file (cm-build output).